charts/testkube-api/executors.json

```diff
@@ -134,6 +134,46 @@
             }
         }
     },
+    {
+        "name": "jmeterd-executor",
+        "executor": {
+            "executorType": "job",
+            "image": "kubeshop/testkube-jmeterd-executor:2f2a937",
+            "command": [
+                "<entryPoint>"
+            ],
+            "args": [
+                "-n",
+                "-j",
+                "<logFile>",
+                "-t",
+                "<runPath>",
+                "-l",
+                "<jtlFile>",
+                "-e",
+                "-o",
+                "<reportFile>",
+                "<envVars>"
+            ],
+            "types": [
+                "jmeter/test"
+            ],
+            "contentTypes": [
+                "string",
+                "file-uri",
+                "git-file",
+                "git-dir",
+                "git"
+            ],
+            "features": [
+                "artifacts"
+            ],
+            "meta": {
+                "iconURI": "jmeter",
```

Comment:
will we use same docs and icon?

Reply:
probably different, but this is not a priority atm